docs/conf.py

```diff
@@ -0,0 +1,281 @@
+
+try:
+    import sphinx_rtd_theme
+except ImportError:
+    sphinx_rtd_theme = None
+
+try:
+    from sphinxcontrib import spelling
+except:
+    spelling = None
+
+from sphinxcontrib.pandoc_markdown import MarkdownParser
+
+# label = subprocess.check_output(["git", "describe", "--tags", "--always"]).strip().decode("utf-8")
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.ifconfig',
+    'sphinx.ext.todo'
+]
+
+if spelling is not None:
+    extensions.append('sphinxcontrib.spelling')
+
+# Add any paths that contain templates here, relative to this directory.
+# templates_path = ['_templates']
+
+# The suffix of source filenames.
+#source_suffix = '.rst'
+source_suffix = ['.rst', '.md']
+
+# The encoding of source files.
+# source_encoding = 'utf-8-sig'
+
+# set ifconfig tags
+if not 'CONTENT_ID_BASE' in os.environ or 'build-' in os.environ['CONTENT_ID_BASE']:
+    # for Nexus previews, gh-pages, and local builds
+    internal = True
+else:
+    # for Nexus publishing
+    internal = False
+
+# The master toctree document.
+master_doc = 'index'
+
+# The builder to use when running via the deconst preparer
+# builder = 'deconst-single'
+builder = 'deconst-serial'
+
+# Exclude content from elastic search index
+deconst_default_unsearchable = True
+
+# linkcheck options
+linkcheck_ignore = ['https://pages.github.rackspace.com*',
+                    'https://pitchfork.eco.rackspace.com*',
+                    'https://rpc1.kubernetes.iad.ohthree.com*',
+                    'https://developer.rackspace.com*']
+linkcheck_anchors = False
+
+# General information about the project.
+project = 'Rackspace RPC rpc-ceph'
+copyright = '2018, Rackspace'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+
+# The short X.Y version.
+try:
+    if 'master' in EventHandlers.CURRENT_VERSION:
+       formatted = 'master'
+    else:
+       formatted = EventHandlers.CURRENT_VERSION
+except TypeError:
+    formatted = '1.0.x'
+
+version = '{}'.format(formatted)
+
+# The full version, including alpha/beta/rc tags.
+release = version
+
+# Global variables that are replaced by the specified value during the build
+# process.
+
+rst_epilog = """
+.. |os-release| replace:: Newton
+.. |os-prev-release| replace:: Mitaka
```

Comment:
Should we define ceph-release variable as luminous?  Is the previous release jewel, luminous or N/A?

Reply:
Done